### DISCUSSION TITLE
How to set the number of samples in the training set and test set during training

### DISCUSSION THREAD

**Discussion by ChengQIAN93** [deepmodeling/deepmd-kit#481] [Q&A] [answered]
Dear all,

I am confused with training set and test set

1. Is the ratio of test set to training set required? Is there a recommendation?  train set : test set = 1:1  or other value ？ 

2. For example, using dpdata to generate set.000 and set.001, is set.001 used in dp train or dp test?

Thank you and looking for your reply. 

Cheng Qian

**Comment by Dankomaister** [comment, marked as answer]
Hi Cheng!

The last set is used as testing set, while the rest sets are used as training sets.
You can find for information about preparing the data here:
https://deepmd.readthedocs.io/en/master/use-deepmd-kit.html#prepare-data

Regarding the ratio between training and testing data, that depends on the size of your dataset
But using around 20% of the data as testing data is a good start.

/Daniel

**Reply by ChengQIAN93** [reply]
Thank you for your reply  👍
